src/views/components/avatarHome/avatarHome.tsx

```diff
@@ -23,65 +23,61 @@ const responsive = {
   },
 };
 
-type AvatarHomeRoutes = {
-  getalluseravatars: string;
-  changeavatar: string;
-  customavatar: string;
-  edit: string;
+const GetLinks = {
+  getalluseravatars: (username: string) =>
+    `/profile/${username}/avatar/getalluseravatars`,
+  changeavatar: (username: string) =>
+    `/profile/${username}/avatar/changeavatar`,
+  customavatar: (username: string) => `/profile/${username}/customavatar`,
+  edit: (username: string) => `/profile/${username}/edit`,
 };
 
 export function AvatarHome() {
-  const [currentResImgLink, setCurrentResImgLink] = useState(
-    '/avatars/base-res.svg',
+  const [username, setUsername] = useState<string | null>(null);
+  const [currentResImgLink, setCurrentResImgLink] = useState<string | null>(
+    null,
   );
-  const [currentSpyImgLink, setCurrentSpyImgLink] = useState(
-    '/avatars/base-spy.svg',
+  const [currentSpyImgLink, setCurrentSpyImgLink] = useState<string | null>(
+    null,
   );
   const [avatarLibrary, setAvatarLibrary] = useState<S3AvatarSet[]>([]);
   const [selectedAvatarSet, setSelectedAvatarSet] =
     useState<S3AvatarSet | null>(null);
   const [isLoading, setIsLoading] = useState<boolean>(true);
-  const [routes, setRoutes] = useState<AvatarHomeRoutes | null>(null);
 
   useEffect(() => {
+    require('../../styles.css');
     require('./styles.css');
     require('react-multi-carousel/lib/styles.css');
 
     // Extract username from route of form '/profile/:profileusername/avatar'
     const match = window.location.pathname.match(/\/profile\/([^\/]+)\/avatar/);
     const extractedUsername = match ? match[1] : '';
-
-    const newRoutes: AvatarHomeRoutes = {
-      getalluseravatars: `/profile/${extractedUsername}/avatar/getalluseravatars`,
-      changeavatar: `/profile/${extractedUsername}/avatar/changeavatar`,
-      customavatar: `/profile/${extractedUsername}/customavatar`,
-      edit: `/profile/${extractedUsername}/edit`,
-    };
-
-    setRoutes(newRoutes);
+    setUsername(extractedUsername);
   }, []);
 
   useEffect(() => {
     setIsLoading(true);
-    if (routes) {
+
+    if (username) {
       void fetchUserAvatarInfo();
     }
 
     async function fetchUserAvatarInfo() {
-      const response = await fetch(routes.getalluseravatars);
+      const response = await fetch(GetLinks.getalluseravatars(username));
       const data: AllAvatarsRouteReturnType = await response.json();
 
-      if (data.currentResLink) {
-        setCurrentResImgLink(data.currentResLink);
-      }
-      if (data.currentSpyLink) {
-        setCurrentSpyImgLink(data.currentSpyLink);
-      }
+      setCurrentResImgLink(
+        data.currentResLink ? data.currentResLink : '/avatars/base-res.svg',
```

Comment:
Let's move `/avatars/base-res.svg` to a const at the top or in a higher level file.